src/main/java/myproject/likelihoodofnaturaldisaster/controller/GeoController.java

```diff
@@ -21,9 +22,10 @@
     @Autowired
     private AlexeyDtoMapper mapper;
 
-
     private static final String GEO_PAGE = "geoPage";
-    public static final String API_KEY = "321454184d08ad9ad1910cf541bab553";
+
+    @Value("${321454184d08ad9ad1910cf541bab553}")
```

Comment:
@katorzhin , WTF???? 
you described this value in app.properties but use the key itself??
BTW I asked you to change only message.properties. Why this file appear in your PR?

Reply:
@GetFire  you asked me to do it earlier, so I did like you told me